src/strategy-factories/get-ipfs-tokenuri/extractor.mjs

```diff
@@ -7,14 +7,10 @@ import logger from "../../logger.mjs";
 import { fileExists } from "../../disc.mjs";
 
 export const getIpfsTokenUriFactory = (props) => {
-  const { strategyName, options, version, schema } = props;
+  const { strategyName, options, version, schema, transformTokenUri } = props;
   const log = logger(strategyName);
 
-  if (env.IPFS_HTTPS_GATEWAY_KEY) {
```

Comment:
Why has this been removed? I think it is needed, isn't it?

Reply:
https://github.com/neume-network/extraction-worker/issues/58

Reply:
Sorry, a little confusion on my part. You are right. This was removed because extraction-worker didn't support header for IPFS.